addons/mail/models/discuss/discuss_channel_member.py

```diff
@@ -149,6 +156,21 @@ def _notify_typing(self, is_typing):
             notifications.append([member.channel_id.uuid, 'discuss.channel.member/typing_status', formatted_member])  # notify livechat users
         self.env['bus.bus']._sendmany(notifications)
 
+    @api.model
+    def _unmute(self):
+        # Unmute notifications for the all the channel members whose mute date is passed.
+        members = self.search([("mute_until_dt", "<=", fields.Datetime.now())])
+        members.write({"mute_until_dt": False})
+        notifications = []
+        for member in members:
+            channel_data = {
+                "id": member.channel_id.id,
+                "model": "discuss.channel",
+                "mute_until_dt": False,
+            }
+            notifications.append((member.partner_id, "mail.record/insert", {"Thread": channel_data}))
```

Comment:
review note: removed the specific notif and just updated channel data for member